### DISCUSSION TITLE
Wallet Address issue

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#463] [Help]
https://github.com/Web3Auth/Web3Auth/tree/master/demo/react-app
I cloned this project in my local machine. It was working fine. I was doing some test with this demo. But suddenly after few hours, when I logged in again after logging out, my wallet address was changed. I also lost some data I had with precious wallet address. Please explain me why is this happening. Is there anything that I still need to understand properly?

**Comment by himanshuchawla009** [comment]
Can you confirm you were using same social login method with same account?

using different social logins methods or accounts will result in different wallet address.

**Reply by KetulCodiste** [reply]
Yes, I was using the same social login.
I logged in through "continue with google" with the same email address.

**Reply by himanshuchawla009** [reply]
You have created one more issue, where you have described that you were using different demos and hence you were getting different accounts. I have answered your question more descriptively here: 
https://github.com/orgs/Web3Auth/discussions/456

**Comment by andrebruch** [comment]
Same issue here!

A user has created a wallet with his email (tkey-auth0-email-passwordless) on 6/13/22.
The same user logged in with his email (tkey-auth0-email-passwordless) on 8/02/22 and created a new wallet.

How can it be that a user with the same login method creates a new wallet instead of logging into his old wallet?

ChainId: 0x38

Used npm packages:
web3@1.7.4
@web3auth/web3auth@1.1.0
@web3auth/openlogin-adapter@1.1.0

**Reply by himanshuchawla009** [reply]
plz refer to the answer given in this discussion: https://github.com/orgs/Web3Auth/discussions/456

**Reply by andrebruch** [reply]
@himanshuchawla009 @shahbaz17 
My case is different and the answer https://github.com/orgs/Web3Auth/discussions/456 or https://web3auth.io/docs/troubleshooting/different-wallet-address-issue does not apply. The app has been in production for months and the same clientId, same aggregateVerifier (tkey-auth0-email-passwordless), same verifier (torus) and same verifierId have always been used.

I have created a separate issue (https://github.com/Web3Auth/Web3Auth/issues/550) / discussion (https://github.com/Web3Auth/Web3Auth/discussions/566).

**Comment by shahbaz17** [comment]
Answered https://github.com/orgs/Web3Auth/discussions/456
